middleware/src/middleware.go

```diff
@@ -354,91 +346,63 @@ func (middleware *Middleware) GetHostname() rpcmessages.GetHostnameResponse {
 	}
 }
 
-// EnableTor enables/disables the tor.service and configures bitcoind and lightningd based on the passed boolean argument
+// EnableTor enables/disables the tor.service and configures bitcoind and lightningd based on the passed ToggleSettingENABLE/DISABLE argument
 // and returns a ErrorResponse indicating if the call was successful.
-func (middleware *Middleware) EnableTor(enable bool) rpcmessages.ErrorResponse {
-	var action string
-	if enable {
-		log.Println("Enabling Tor via the config script")
-		action = enableAction
-	} else {
-		log.Println("Disabling Tor via the config script")
-		action = disableAction
-	}
+func (middleware *Middleware) EnableTor(toggleAction rpcmessages.ToggleSetting) rpcmessages.ErrorResponse {
+	log.Println("Executing " + toggleAction + " Tor via the config script")
```

Comment:
That should probably be a `log.Printf` for consistency? *(multiple concurrencys)*
```suggestion
	log.Printf("Executing a '%s Tor' via the config script.\n",  toggleAction)
```

Reply:
Sure. I suppose you mean that for all the methods in this PR then?

Reply:
Yes